libraries/botframework-connector/.gitignore

```diff
@@ -1,5 +1,5 @@
-/**/node_modules
-/**/.vscode
-/**/lib/*
-coverage
 .nyc_output
+.vscode
```

Comment:
I think we can move away from having these individual .gitignores, but that can be tracked and performed in a later work item